client/src/App.js

```diff
@@ -2,22 +2,16 @@ import './App.css';
 
 import React, { useState } from 'react';
 
+import { PAGE_MAP } from './constants/header_constant';
```

Comment:
Minor nit: the filename `header_constants` would probably be better because I see you have more than one constant exported.